Need to add null check on UsbManager

An user reported this on Telegram. From the crash log we suspected he is using Amaze on Windows 11's Android subsystem, which is not officially supported.

Although Amaze's focus is on true Android, it would be nice if we can handle situations like this gracefully - like a Toast to notify user that we cannot enumerate USB devices.

## Issue explanation (write below this line)



## Exception
* __App Name:__ Amaze File Manager
* __Package:__ com.amaze.filemanager
* __Version:__ 3.6.5
* __User Action:__ UI Error
* __Request:__ Application crash
* __OS:__ Linux Android 11 - 30
* __Device:__ windows_x86_64
* __Model:__ Subsystem for Android(TM)
* __Product:__ windows_x86_64
<details><summary><b>Crash log </b></summary><p>

```
java.lang.RuntimeException: Unable to resume activity {com.amaze.filemanager/com.amaze.filemanager.ui.activities.MainActivity}: java.lang.NullPointerException: null cannot be cast to non-null type android.hardware.usb.UsbManager
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4438)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4470)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7664)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.NullPointerException: null cannot be cast to non-null type android.hardware.usb.UsbManager
	at com.amaze.filemanager.utils.OTGUtil.getMassStorageDevicesConnected(OTGUtil.kt:198)
	at com.amaze.filemanager.ui.activities.MainActivity.updateUsbInformation(MainActivity.java:1290)
	at com.amaze.filemanager.ui.activities.MainActivity.onResume(MainActivity.java:1282)
	at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1456)
	at android.app.Activity.performResume(Activity.java:8129)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4428)
	... 11 more

```
</details>
<hr>

--- a/app/src/main/java/com/amaze/filemanager/utils/OTGUtil.kt
+++ b/app/src/main/java/com/amaze/filemanager/utils/OTGUtil.kt
@@ -195,8 +195,8 @@ object OTGUtil {
     fun getMassStorageDevicesConnected(
         context: Context
     ): List<UsbOtgRepresentation> {
-        val usbManager = context.getSystemService(Context.USB_SERVICE) as UsbManager
-        val devices = usbManager.deviceList
+        val usbManager = context.getSystemService(Context.USB_SERVICE) as? UsbManager
+        val devices = usbManager?.deviceList ?: mapOf()
         return devices.mapNotNullTo(
             ArrayList(),
             { entry ->

